bond_calculations.py

```diff
@@ -344,6 +343,12 @@ def main():
 
     # Upload all bonds metadata to socrata
     df = get_data(client, "all_bonds_program_names")
+
+    # Join in the appropriation totals
+    app = get_data(client, "all_bonds_appropriations")
+    app = app.pivot_table(index="dashboard_deptfundprogact", values="amount", aggfunc=sum)
```

Comment:
rad - i only know of pivot tables from excel so it was cool to read about them in pandas for aggregations like this. TIL!